test/__snapshots__/storyshots.test.js.snap

```diff
@@ -7169,7 +7176,9 @@ exports[`RippleStoryshots Molecules/Form Form 1`] = `
                       <span
                         class="multiselect__option"
                       >
-                        No elements found. Consider changing the search query.
+                        <span>
```

Comment:
Not sure why - but this reverts even on master branch (before these changes are implemented).